config.json

```diff
@@ -134,10 +134,10 @@
         "storagePrefix": "session/"
     },
     "server": {
-        "baseUrl": "https://api.opentoken.io",
+        "baseUrl": "http://localhost",
```

Comment:
Do you need to put this back to api.opentoken.io for deployment, or do you upload a different config when deploying?